include/bluetooth/iso.h

```diff
@@ -74,6 +74,19 @@ enum {
 	BT_ISO_CHAN_QOS_INOUT
 };
 
+/** @brief ISO slave clock accuracy in ppm (parts per million) */
+enum {
+	BT_ISO_SCA_UNKNOWN = 0,
```

Comment:
It shouldn't be bound to ISO.
Sleep clock accuracy could used for connections, and it seems periodic advertising as well.

Reply:
Could be added as information for ACL connections, (sca in connection complete event).
Also there is the "4.6.25 Sleep Clock Accuracy Updates" feature.
So I don't think having ISO in the namespace is wise.

Reply:
@joerchan Where do you suggest we put it then? bluetooth.h? gap.h? conn.h?

Reply:
Moved